### PR TITLE
docs: 3.19.3 change-log

### PR DESCRIPTION
**Pull request by ikobe-zz**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

-----

- 📝 增加 DatePicker/RangePicker 指定 `mode` 后无法选择的文档说明。[#16984](https://github.com/ant-design/ant-design/pull/16984)
- 🐞 修复 Breadcrumb 的 `validateDOMNesting` 警告信息。[#16929](https://github.com/ant-design/ant-design/pull/16929)
- 🐞 修复 Breadcrumb 选中子路由时浏览器路径问题。[#16885](https://github.com/ant-design/ant-design/pull/16885) [@haianweifeng](https://github.com/haianweifeng)
- 🐞 修复 InputNumber 设置 `number` 类型时会展示原生按钮的问题。[#16926](https://github.com/ant-design/ant-design/pull/16926)
- 🐞 修复 Transfer 在自定义列表为空时展示 Empty 样式。[#16925](https://github.com/ant-design/ant-design/pull/16925)
- 🐞 **修复 Table 头部多余的垂直滚动条样式。**[#16950](https://github.com/ant-design/ant-design/pull/16950)
- 🐞 修复 Table 的 `rowSelection.getCheckboxProps()` 在没有返回值时报错的问题。[#15224](https://github.com/ant-design/ant-design/pull/15224)
- 🐞 修复 Firefox 的 Table 丢失 `border-radius` 样式问题。[#16957](https://github.com/ant-design/ant-design/pull/16957)
- 🐞 修复 Table 当 `title` 和 `rowSelection` 同时指定时在 Chrome 下滚动条异常的问题。[#16934](https://github.com/ant-design/ant-design/pull/16934)
- 🐞 修复 Divider `orientation="center"` 时样式错位的问题。[#16988](https://github.com/ant-design/ant-design/pull/16988)
- 🐞 修复 Cascader 搜索时不支持空格输入的问题。[#16918](https://github.com/ant-design/ant-design/pull/16918) [@Durisvk](https://github.com/Durisvk)
- 🐞 修复部分组件的西班牙语言翻译。[#17002](https://github.com/ant-design/ant-design/pull/17002) [@morellan](https://github.com/morellan)
- TypeScript
  - 🐞 修复 Upload 的 `RcFile` 类型定义。[#16851](https://github.com/ant-design/ant-design/pull/16851)
  - ⚡️ 导出 Typography 中 `TextProps` 的类型定义。[#17003](https://github.com/ant-design/ant-design/pull/17003) [@Jarvis1010](https://github.com/Jarvis1010)

-----

- 📝 Add FAQ for DatePicker/RangePicker with `mode` cannot be selected. [#16984](https://github.com/ant-design/ant-design/pull/16984)
- 🐞 Fix Breadcrumb validateDOMNesting warning. [#16929](https://github.com/ant-design/ant-design/pull/16929)
- 🐞 Fix Breadcrumb path error when `children` is selected. [#16885](https://github.com/ant-design/ant-design/pull/16885) [@haianweifeng](https://github.com/haianweifeng)
- 🐞 Fix InputNumber with `number` type display native spinner. [#16926](https://github.com/ant-design/ant-design/pull/16926)
- 🐞 Fix Transfer render Empty when customize without data. [#16925](https://github.com/ant-design/ant-design/pull/16925)
- 🐞 **Fix Table header extra vertical scrollbar problem.** [#16950](https://github.com/ant-design/ant-design/pull/16950)
- 🐞 Fix Table miss `border-radius` in Firefox. [#16957](https://github.com/ant-design/ant-design/pull/16957)
- 🐞 Fix Table error when `rowSelection.getCheckboxProps()` has no return value. [#15224](https://github.com/ant-design/ant-design/pull/15224)
- 🐞 Fix Table abnormal scrollbar in Chrome when using `title` and `rowSelection`.
 [#16934](https://github.com/ant-design/ant-design/pull/16934)
- 🐞 Fix Divider `orientation="center"` style. [#16988](https://github.com/ant-design/ant-design/pull/16988)
- 🐞 Fix Cascader error when type space. [#16918](https://github.com/ant-design/ant-design/pull/16918) [@Durisvk](https://github.com/Durisvk)
- 🐞 Fix missing spanish translations. [#17002](https://github.com/ant-design/ant-design/pull/17002) [@morellan](https://github.com/morellan)
- TypeScript
  - 🐞 Fix Upload `RcFile` definition. [#16851](https://github.com/ant-design/ant-design/pull/16851)
  - ⚡️ Export `TextProps` type in Typography. [#17003](https://github.com/ant-design/ant-design/pull/17003) [@Jarvis1010](https://github.com/Jarvis1010)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
 
+## 3.19.3
+
+`2019-06-06`
+
+- 🐞 Fixed Breadcrumb validateDOMNesting warning. [16929](https://github.com/ant-design/ant-design/pull/16929)
+- 🐞 Fix InputNumber with `number` type display native spinner. [16926](https://github.com/ant-design/ant-design/pull/16926)
+- 🐞 Fix Transfer render Empty when customize without data. [16925](https://github.com/ant-design/ant-design/pull/16925)
+- 🐞 Fixed Table header extra vertical scrollbar problem. [16950](https://github.com/ant-design/ant-design/pull/16950)
+- 🐞 Fix Table miss `border-raidus` in Firefox. [16957](https://github.com/ant-design/ant-design/pull/16957)
```
Comment (ztplz) [inline CHANGELOG.en-US.md]: `raidus` => `radius`

**Review by ztplz — APPROVED** [review]

**Comment by ztplz** [comment]
不是还没到周末吗

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
+- 🐞 Fixed Divider `orientation="center"` style. [16988](https://github.com/ant-design/ant-design/pull/16988)
+- 🐞 Fix Table error when `rowSelection.getCheckboxProps()` has no return value. [15224](https://github.com/ant-design/ant-design/pull/15224)
+- 🐞 Fix Breadcrumb path error when children is selected. [16885](https://github.com/ant-design/ant-design/pull/16885)
+- 🐞 Fix Cascader error when type space. [16918](https://github.com/ant-design/ant-design/pull/16918)
+- 🐞 Fix Table abnormal scrollbar in Chrome when using `title` and `rowSelection`.
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 相同组件放在一起。

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
+ [16934](https://github.com/ant-design/ant-design/pull/16934)
+- 📝 Add FAQ for DatePicker/RangePicker with `mode` cannot be selected. [16984](https://github.com/ant-design/ant-design/pull/16984)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 这个放最上面吧。

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -15,6 +15,25 @@ timeline: true
...
 ---
 
+## 3.19.3
+
+`2019-06-06`
+
+- 🐞 Fixed Breadcrumb validateDOMNesting warning. [16929](https://github.com/ant-design/ant-design/pull/16929)
+- 🐞 Fix InputNumber with `number` type display native spinner. [16926](https://github.com/ant-design/ant-design/pull/16926)
+- 🐞 Fix Transfer render Empty when customize without data. [16925](https://github.com/ant-design/ant-design/pull/16925)
+- 🐞 Fixed Table header extra vertical scrollbar problem. [16950](https://github.com/ant-design/ant-design/pull/16950)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: Fixed Fix 统一

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,24 @@ timeline: true
...
+- 🐞 修复 InputNumber 设置 `number` 类型时会展示原生按钮的问题。[16926](https://github.com/ant-design/ant-design/pull/16926)
+- 🐞 修复 Transfer 在自定义列表为空时展示 Empty 样式。[16925](https://github.com/ant-design/ant-design/pull/16925)
+- 🐞 修复 Table 头部多余的垂直滚动条样式。[16950](https://github.com/ant-design/ant-design/pull/16950)
+- 🐞 修复 Firefox 的 Table 丢失 `border-raidus` 样式问题。[16957](https://github.com/ant-design/ant-design/pull/16957)
+- 🐞 修复 Divider `orientation="center"` 时样式错位的问题。[16988](https://github.com/ant-design/ant-design/pull/16988)
+- 🐞 修复 Table 的 `rowSelection.getCheckboxProps()` 在没有返回值时报错的问题。[15224](https://github.com/ant-design/ant-design/pull/15224)
+- 🐞 修复 Breadcrumb 选中子路由时浏览器路径问题。[16885](https://github.com/ant-design/ant-design/pull/16885)
+- 🐞 修复 Cascader 搜索时不支持空格输入的问题。[16918](https://github.com/ant-design/ant-design/pull/16918)
+- 🐞 修复 Table 当 `title` 和 `rowSelection` 同时指定时在 Chrome 下滚动条异常的问题。[16934](https://github.com/ant-design/ant-design/pull/16934)
+- 📝 增加 DatePicker/RangePicker 指定 mode 后无法选择的文档说明。[16984](https://github.com/ant-design/ant-design/pull/16984)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: mode 加 `

File: CHANGELOG.en-US.md (hunk 4 above)
Comment (afc163) [reply CHANGELOG.en-US.md]: 这个可以加粗。

**Comment by afc163** [comment]
image

https://github.com/ant-design/ant-design/wiki/%E8%BD%AE%E5%80%BC%E8%A7%84%E5%88%99%E5%92%8C%E7%89%88%E6%9C%AC%E5%8F%91%E5%B8%83%E6%B5%81%E7%A8%8B

**Comment by ikobe** [comment]
@afc163 fixed

**Comment by afc163** [comment]
又合并了几个 PR

**Comment by ztplz** [comment]
其实应该用 `fix` 的，虽然是过去完成的，但是这个意思是表示一件很通常的事，我修复了xxx，而不是我已经修复了xxx。

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -15,6 +15,25 @@ timeline: true
 
 ---
 
+## 3.19.3
+
+`2019-06-06`
+
+- 📝 Add FAQ for DatePicker/RangePicker with `mode` cannot be selected. [16984](https://github.com/ant-design/ant-design/pull/16984)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: issue 链接没加 # 号

**Comment by afc163** [comment]
> 又合并了几个 PR

**Comment by afc163** [comment]
- https://github.com/ant-design/ant-design/pull/17003
- https://github.com/ant-design/ant-design/pull/17002

这两个可以写一下

**Comment by afc163** [comment]
PR 要附上外部贡献者的名字。

**Comment by ikobe** [comment]
Done

File: CHANGELOG.zh-CN.md [hunk 7]
```diff
@@ -15,6 +15,26 @@ timeline: true
...
+- 🐞 修复 Breadcrumb 选中子路由时浏览器路径问题。[#16885](https://github.com/ant-design/ant-design/pull/16885)
+- 🐞 修复 InputNumber 设置 `number` 类型时会展示原生按钮的问题。[#16926](https://github.com/ant-design/ant-design/pull/16926)
+- 🐞 修复 Transfer 在自定义列表为空时展示 Empty 样式。[#16925](https://github.com/ant-design/ant-design/pull/16925)
+- 🐞 **修复 Table 头部多余的垂直滚动条样式。**[#16950](https://github.com/ant-design/ant-design/pull/16950)
+- 🐞 修复 Table 的 `rowSelection.getCheckboxProps()` 在没有返回值时报错的问题。[#15224](https://github.com/ant-design/ant-design/pull/15224)
+- 🐞 修复 Firefox 的 Table 丢失 `border-radius` 样式问题。[#16957](https://github.com/ant-design/ant-design/pull/16957)
+- 🐞 修复 Table 当 `title` 和 `rowSelection` 同时指定时在 Chrome 下滚动条异常的问题。[#16934](https://github.com/ant-design/ant-design/pull/16934)
+- 🐞 修复 Divider `orientation="center"` 时样式错位的问题。[#16988](https://github.com/ant-design/ant-design/pull/16988)
+- 🐞 修复 Cascader 搜索时不支持空格输入的问题。[#16918](https://github.com/ant-design/ant-design/pull/16918)
+- 🐞 修复 Typography 警告文案的英文翻译。[#17003](https://github.com/ant-design/ant-design/pull/17003)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 这个应该是 TextProps 这个属性相关

**Comment by ikobe** [comment]
@afc163 fixed

File: CHANGELOG.en-US.md [hunk 8]
```diff
@@ -15,6 +15,27 @@ timeline: true
 
 ---
 
+## 3.19.3
+
+`2019-06-06`
+
+- 📝 Add FAQ for DatePicker/RangePicker with `mode` cannot be selected. [#16984](https://github.com/ant-design/ant-design/pull/16984)
+- 🐞 Fix Breadcrumb validateDOMNesting warning. [#16929](https://github.com/ant-design/ant-design/pull/16929)
+- 🐞 Fix Breadcrumb path error when children is selected. [#16885](https://github.com/ant-design/ant-design/pull/16885) [@haianweifeng](https://github.com/haianweifeng)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: `children`

**Pull request merged**